server/container/spring/src/main/java/org/apache/james/container/spring/tool/James23ImporterManagement.java

```diff
@@ -69,23 +56,8 @@ public void importMailsFromJames23(String james23MailRepositoryPath) throws Exce
         try {
             james23Importer.importMailsFromJames23(james23MailRepositoryPath);
         } catch (Exception e) {
-            e.printStackTrace();
+            LOGGER.error("Error while importing mail", e);
```

Comment:
This code is never used, and not supported for know.

This change is intended to **report issue** *without changing behaviour*